machine-learning/machine-learning-quiz.md

```diff
@@ -282,3 +282,46 @@
 - [ ]  It would increase the speed of the appliance.
 
 [✔]
+#### QTraining1. You work for a hospital that is tracking the community spread of a virus. The hospital created a smartwatch app that uploads body temperature data from hundreds of thousands of participants. What is best technique to analyze the data?
+- [] Use reinforcement learning to reward the system when a new person participates
+- [] Unsupervised machine learning to cluster together people based on patterns the machine discovers
+- [] Supervised machine learning to sort people by demographic data
+- [x] supervised ml to classify people by body temperature
+
+#### QTraining2. Man of the advances in ml have come from improved
+- [] statistics
+- [x] structured data
+- [] availability
+- [] algorithms
```

Comment:
@tik9  Timo just small doubt is this question from machine learning or ? can you please double check
thanks

Reply:
maybe just change `#### QTraining2.` to real numbers